python/samseg/lta.py

```diff
@@ -103,49 +115,188 @@ def read(self, filename):
             self.srcfile = self.srcmri.read(content)
             self.dstfile = self.dstmri.read(content)
             self.subject = parse_expected_string_no_equals('subject', content)
-
+        return self
 
     def write(self, filename):
         with open(filename, 'w') as file:
             file.writelines('\n'.join([
-                '# transform file {0}'.format(filename),
-                '# created by LTA.write',
-                'type     = %d' % self.type,
-                'nxforms  = %d' % self.nxforms,
-                'mean     = {}'.format(' '.join([str(val) for val in self.mean])),
-                'sigma     = %d' % self.sigma,
-                ' '.join(str(val) for val in self.dims),
-                ] + self.transformLines() + [
-                ] + self.srcmri.formatted_lines('src', self.srcfile) + [
-                ] + self.dstmri.formatted_lines('dst', self.dstfile) + [
-            ' '.join(['subject', self.subject or 'unknown']),
-            ]))
+                                          '# transform file {0}'.format(filename),
```

Comment:
This needs some breaking up. I would also recommend `terminaltables` for displaying pretty tabular data on the command line.

Reply:
Breaking up for readability is feasible. However this data goes into a file that is to be read by unknown parsers, so current formatting should be maintained. Over all, however, the project should head towards using nice standard formatters like `terminaltables` for UI and something standard, like json, for inter process communication.

Reply:
that makes sense. I did not realize this was going to a file.